Formalisation(shacl)/Core/Example-Data/example-catalog.ttl

```diff
@@ -1,12 +1,31 @@
 @prefix dcat: <http://www.w3.org/ns/dcat#> .
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
```

Comment:
The dcterms namespace is already defined on line 2, this is an unnecessary duplicate.

```suggestion
```